### DISCUSSION TITLE
大佬，我在B站获取元素的时候，插件为何会出现两次加载的情况！

### DISCUSSION THREAD

**Discussion by hackhase** [lisonge/vite-plugin-monkey#133] [General]
大佬，我在B站获取元素的时候，第一次加载的时候可以获取到元素，第二次加载的时候，没法获取元素了，这个是什么情况的？
console.log(document.getElementsByClassName('bpx-player-ctrl-quality-text'));
![image](https://github.com/lisonge/vite-plugin-monkey/assets/109451138/7f1b9316-9ec5-42c2-9f49-fccd568ba783)

**Comment by Dave-12138** [comment]
也许只是因为，页面内有个iframe，而在iframe的src符合你的脚本适用范围时，脚本会在iframe中运行？<img src="https://bbs-static.miyoushe.com/static/2024/06/19/c638e43d0e2701e8f3e29a82dfd659fe_7613886348305980822.png" alt="流萤 没钱" width="75" >
